client/src/charts/canada/CanadaD3Component.js

```diff
@@ -237,9 +229,12 @@ export class CanadaD3Component {
 
         d3.select(this)
           .append("p")
-          .style("margin-bottom", "0px")
-          .html(formatter(last_year_data[prov_key] || 0));
+          .attr("class", "label-value")
+          .style("margin-bottom", "0px");
       });
+    html.selectAll("p.label-value").each(function (prov_key, i) {
```

Comment:
Just a minor thing, eslint lodash has a preference to use .forEach since .each is an alias and using .forEach only will be consistent once the merge is completed. (If you don't like this preference, we can disable it. Just let me know!)

Reply:
Ahh I think this is `each` from d3 so it should be irrelevant!